titan-server/src/accounts/acl.rs

```diff
@@ -2,35 +2,37 @@ use diesel::{prelude::*, MysqlConnection};
 
 use crate::models;
 use crate::schema;
-
-pub fn get_user_acl(wcf_user_id: i32, wcf_db: &MysqlConnection) -> QueryResult<Vec<models::WcfAclOption>> {
-    let mut acl_option_ids = schema::wcf1_user_to_group::table
-        .inner_join(schema::wcf1_acl_option_to_group::table.on(
-            schema::wcf1_acl_option_to_group::group_id.eq(
-                schema::wcf1_user_to_group::group_id
+use crate::models::WcfUserOptionValue;
+use diesel::mysql::Mysql;
+
+pub fn get_user_acl(wcf_user_id: i32, wcf_db: &MysqlConnection) -> QueryResult<Vec<models::WcfUserGroupOption>> {
```

Comment:
I noticed this query only checks group permissions. Do we have any use cases where ACL options will need to be assigned to the user and not a group?

Reply:
No. There won't be any cases like that. ACL options are applied per-user, per-forum. Group permissions apply system-wide.